Cilsil/Program.cs

```diff
@@ -118,19 +116,32 @@ private static void Main(string[] args)
         /// Executes the translation.
         /// </summary>
         /// <param name="paths">The paths.</param>
-        /// <param name="printprocs">The printprocs.</param>
-        /// <param name="debug">The debug flag.</param>
-        /// <returns></returns>
-        public static (Cfg, TypeEnvironment) ExecuteTranslation(string[] paths,
-                                                                string printprocs = null,
-                                                                bool debug = false)
+        /// <param name="extensionProgress">If <c>true</c>, periodically write progress to 
+        /// console if the input binaries are adequately large.</param>
+        /// <returns>The computed cfg and type environment.</returns>
+        public static (Cfg, TypeEnvironment) ExecuteTranslation(
+            string[] paths, bool extensionProgress = false)
         {
             var assemblies = GetAssemblies(paths);
 
+            var totalSize = assemblies.Select(p =>
```

Comment:
Why not return `totalSize` in `GetAssemblies()` function instead of reiterating again?

Reply:
Done